src/plone/restapi/tests/test_services_querystring.py

```diff
@@ -93,7 +93,7 @@ def test_endpoint_inlines_vocabularies(self):
             u"published": {u"title": u"Published with accent \xe9 [published]"},
             u"visible": {u"title": u"Public draft [visible]"},
         }
-        self.assertEqual(expected_vocab_values, idx["values"])
+        self.assertDictContainsSubset(expected_vocab_values, idx["values"])
```

Comment:
This is deprecated: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertDictContainsSubset

Reply:
Thank you for reviewing. I'll fix this as soon as the main PR https://github.com/plone/plone.app.discussion/pull/165 is reviewed and approved.